src/mixins/input-styles/_input-styles.scss

```diff
@@ -0,0 +1,37 @@
+@mixin input-styles() {
+  @include text(body1);
+
+  line-height: 1;
+  width: 100%;
+  padding: spacing(3, 4);
+  border: 1px solid color(elephant);
+  background-color: color(snow);
+
+  &--focus,
+  &:focus {
+    outline: none;
+    border-color: color(burnt-orange);
+  }
+
+  &--disabled,
+  &:disabled {
+    color: color(elephant);
+    border-color: color(elephant);
+    background-color: color(noise);
+  }
+
+  &--valid {
+    border-color: color(money-good);
+  }
+
+  &--error {
+    &,
```

Comment:
What does the solo & here do?

Reply:
Ensures that these styling overrides the resting state in addition to the focused state